File: 백준/Bronze/2562. 최댓값/최댓값.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
let input = fs.readFileSync(filePath).toString().split("\n");

let max = Number(input[0])
let index = 0
for(let i = 1; i < input.length; i++){
    const cur = Number(input[i])
    if(max < cur){
      max = cur
      index = i
    }
}

console.log(max, index+1 )

File: 백준/Bronze/2566. 최댓값/최댓값.py
A =[]
farr = list(list(map(int,input().split()))for _ in range(9))
for i in farr:
 A.append(max(i))
print(max(A))
for i in range(len(farr[0])):
 for j in range(9):
  if farr[i][j] == max(A): 
   print(i+1, j+1)
   break